server/locale/DE/GPL/styles/flex/ratio--1x4.css

```
@@ -0,0 +1,50 @@
+@media (min-aspect-ratio: 10/41) {
+    .message__content {
+        display: grid;
+        grid-template-columns: 1fr;
+        /*grid-template-rows: 1fr 4fr;*/
+        /*grid-template-rows: 36px 1fr;*/
+        grid-template-rows: 36px 3fr 1fr 36px;
+        align-items: center;
+    }
+    .message__logo-container {
+        align-self: start;
+        width: 100%;
+        margin-top: 3%;
+        margin-bottom: 0;
+    }
+    .message__messaging {
+        flex-direction: column;
+        justify-content: center;
+    }
+    .message__headline {
+        font-size: 3.5vh;
+        line-height: 1.8rem;
+        margin-bottom: 10%;
+    }
+    .message__sub-headline {
+        display: none;
+    }
+    .message__disclaimer {
+        font-size: 0.9rem;
+    }
+    .message__headline .tag--xsmall {
+        display: none;
+    }
+    .message__headline span:nth-of-type(1) {
+        margin-right: -6px;
+    }
+}
+@media (min-aspect-ratio: 10/41) and (max-height: 320px) {
+    .message__content {
+        /*grid-template-rows: 1fr 4fr;*/
+        grid-template-rows: 36px 1fr;
+        /*grid-template-rows: 36px 3fr 1fr 36px;*/
+    }
+    .message__headline {
+        font-size: 1.4rem;
+    }
+    .message__disclaimer {
+        font-size: 0.9rem;
+    }
+}
```

Here's a couple of media queries we can replace the one there now with that will make the look match our other flex messages a bit more as they shrink down.
```suggestion
@media (min-height: 500px) {
    .message__headline {
        font-size: 1.7rem;
    }
}
@media (aspect-ratio: 1/2) {
    .message__messaging {
        transform: translateY(-40px);
    }
}
```